### PR TITLE
Release share2nightscout-bridge 0.2.0

### PR DESCRIPTION
**Pull request by PieterGit**
Changes:
- first release since @bewest. share2nightscout-bridge is now temporary maintained by @sulkaharo and @PieterGit because it's a Nightscout core plugin
- formalize the changes from `wip/generalize` to a released version
- change from `wip/generalize` to `dev` branch, just like oref0 and Nightscout do
- https://github.com/nightscout/share2nightscout-bridge/pull/33 Check res before checking res.statusCode to prevent Nightscout server crashes in case Dexcom server does not respond
- upgrade `request` package
- upgrade node to Node 8 or Node 10 LTS
- publish on npm.org https://www.npmjs.com/package/share2nightscout-bridge . I will publish as a  final `0.2.0` if I get confirmation that https://github.com/nightscout/cgm-remote-monitor/pull/4175 works within Nightscout.

### CODE REVIEW

**Comment by PieterGit** [comment]
@scottleibrand @bewest @veryfancy @ruuddotorg @mayaalinder @kenstack @unsoluble @sulkaharo 
Can you review these changes? I'm about to do a npm release of this share2nightscout version and integrate it into Nightscout, so we can release it with Nightscout 0.11 release.

If anybody wants to take ownership of this project please do. I will do the 0.2.0 release, because I want to get rid of the branched version within Nightscout and want to use npm packages as Nightscout dependencies. I'm not a share2nightscout bridge user, so I need your help to test it and approve it. 

If nobody wants to take ownership now then please append your names a potential developer or tester for share2nightscout bridge plugin with a PR to https://github.com/nightscout/cgm-remote-monitor/blob/dev/CONTRIBUTING.md#plugin-contributors

In a future Nightscout release I would only like to ship maintained plugins with Nightscout, so please step up. There has not been much activity in this GitHub lately, so maintaining this plugin should not be a lot of work.

**Comment by PieterGit** [comment]
Please test https://github.com/nightscout/cgm-remote-monitor/pull/4175

File: index.js [hunk 1]
```diff
@@ -28,13 +28,24 @@ var crypto = require('crypto');
 
 
 // Defaults
+var server = "share1.dexcom.com"
```
Comment (sulkaharo) [inline index.js]: the other bits of code uses semicolons to terminate the lines, could use it here too

File: index.js [hunk 2]
```diff
@@ -28,13 +28,24 @@ var crypto = require('crypto');
...
+var bridge = readENV('BRIDGE_SERVER')
+    if (bridge && bridge.indexOf(".") > 1) {
+    server = bridge;
+   } 
+    else if (bridge && bridge === 'EU') {
+        server = "shareous1.dexcom.com";
+    } else {
+        server = "share1.dexcom.com";
```
Comment (sulkaharo) [inline index.js]: unnecessary assignment

Comment (unsoluble) [reply index.js]: Could you elaborate?

Also, any reason why we couldn’t relax the strict comparison on “EU”? Fairly common problem is folks entering this value in different case.

(I think the case sensitivity should probably be relaxed *everywhere*, but might as well start here while we’re in it...)

Comment (sulkaharo) [reply index.js]: The value is already assigned above, there’s no need to re-assign the same value again

File: index.js (hunk 1 above)
Comment (PieterGit) [reply index.js]: fixed

File: index.js (hunk 2 above)
Comment (PieterGit) [reply index.js]: fixed

**Comment by PieterGit** [comment]
Tested by @unsoluble, see https://gitter.im/nightscout/public?at=5c2d44965a0a8058be1c74a9
Tested by @veryfancy , see https://gitter.im/nightscout/public?at=5c2d58dd8dafa715c7329164

**Pull request merged**